hashintel/hash
# prevent paragraph from being ignored in bolded and italics paragraphs

## 🌟 What is the purpose of this PR?

In the hash.dev blog mdx docs the spacing of bold or italic paragraphs is wrong:

![image](https://user-images.githubusercontent.com/37453800/181729040-20c5dc91-8d30-4757-aad5-7ffe4a9022f5.png)


## 🔗 Related links

- [Asana task](https://app.asana.com/0/1200211978612931/1202405291900008/f) _(internal)_

## 🔍 What does this change?

Our pages weren't rendering paragraphs when they had anything but text inside them, instead just rendering directly what was inside them. ~~Added checks to prevent this from happening when `<strong>` or `<em>` are inside a paragraph.
Maybe the check can be removed entirely, but I'm not sure why it was there in the first place (in BP the check isn't there for paragraphs).~~ Removed the check.

## ❓ How to test this?

1.  Make a whole paragraph bold by wrapping it inside `**` or italics by wrapping it inside `*`;
1.  Check if the paragraph is bold or italicized and if the spacing is correct.

## 📹 Demo

![image](https://user-images.githubusercontent.com/37453800/181729759-68ff77b3-f0ec-4084-8e5c-3d1c034159f2.png)
